internal/http/parser/invalid_errors.yaml

```diff
@@ -0,0 +1,55 @@
+# Protocol Errors
+
+- input: "HTTP/3.0 200 OK"
+  error: |
+    Received: "HTTP/3.0 200 OK"
+               ^ error
+    Error: Expected 'HTTP/1.1', Received: "HTTP/3.0"
+
+- input: "HTTP/1.0_200_OK"
+  error: |
+    Received: "HTTP/1.0_200_OK"
+               ^ error
+    Error: Expected 'HTTP/1.1', Received: "HTTP/1.0"
+
+- input: "HTPP/1.0 200 OK"
+  error: |
+    Received: "HTPP/1.0 200 OK"
+               ^ error
+    Error: Expected 'HTTP/1.1', Received: "HTPP/1.0"
+
+- input: "HTTP|1.0 200 OK"
+  error: |
+    Received: "HTTP|1.0 200 OK"
+               ^ error
+    Error: Expected 'HTTP/1.1', Received: "HTTP|1.0"
+
+- input: "HTTP//1.0 200 OK"
+  error: |
+    Received: "HTTP//1.0 200 OK"
+               ^ error
+    Error: Expected 'HTTP/1.1', Received: "HTTP//1."
+
+- input: "HTTP/1 200 OK"
+  error: |
+    Received: "HTTP/1 200 OK"
+               ^ error
+    Error: Expected 'HTTP/1.1', Received: "HTTP/1 2"
+
+- input: "HTTP/1.1 ZOO OK"
+  error: |
+    Received: "HTTP/1.1 ZOO OK"
+                        ^ error
+    Error: Invalid status-code field, Expected integer value, Received: ZOO
```

Comment:
Should I put this in quotes too ?
It should be an integer, so quoting it might be a bit misleading.